common/src/main/java/com/wynntils/features/utilities/TranscribeMessagesFeature.java

```diff
@@ -144,9 +157,23 @@ private StyledText getStyledTextWithTranscription(
                         defaultColor);
             }
 
+            Style style;
+
+            if ((transcribeGavellian || transcribeWynnic) && !showTooltip.get()) {
+                style = Style.EMPTY.withHoverEvent(new HoverEvent(
+                        HoverEvent.Action.SHOW_TEXT,
+                        Component.translatable("feature.wynntils.transcribeMessages.transcribedFrom")));
```

Comment:
Maybe also append the original wynnic/gavellian text here? (And change the i18n to end in a colon or something like that)